### ISSUE TITLE
Lab Room is null if the lab is in the same room as the lecture.

### ISSUE THREAD

**Issue by dragon0** [SaddlebackCSS/class-list-extractor#3]
See eg the output for Homi's 1A class.

```json
{
    "instructor": "H. Bodhanwala",
    "lab": {
        "day": "M W",
        "room": null,
        "time": "9:30AM - 10:15"
    },
    "lecture": {
        "day": "M W",
        "room": "SM 122",
        "time": "8:00AM - 9:15"
    },
    "number": "80485",
    "status": "Open"
}
```

@rbaker26

**Comment by rbaker26** [comment]
I submitted a PR to deal with this bug

**Comment by dragon0** [comment]
Fixed by #5
